src/components/Proposal/ProposalActions.tsx

```diff
@@ -0,0 +1,86 @@
+import React from 'react'
+
+import useFormatMessage from 'decentraland-gatsby/dist/hooks/useFormatMessage'
+import { Button } from 'decentraland-ui/dist/components/Button/Button'
+
+import { ProposalAttributes } from '../../entities/Proposal/types'
+import { ProposalStatus } from '../../entities/Proposal/types'
+import { ProposalPageState } from '../../pages/proposal'
+
+type ProposalActionsProps = {
+  isOwner: boolean
+  isCommittee: boolean
+  deleting: boolean
+  updatingStatus: boolean
+  proposal: ProposalAttributes | null
+  updatePageState: (newState: Partial<ProposalPageState>) => void
+}
+
+export default function ProposalActions({
+  proposal,
+  isOwner,
+  isCommittee,
+  deleting,
+  updatingStatus,
+  updatePageState,
+}: ProposalActionsProps) {
+  const t = useFormatMessage()
+
+  return (
+    <>
+      {(isOwner || isCommittee) && (
+        <Button
+          basic
+          fluid
+          loading={deleting}
+          disabled={proposal?.status !== ProposalStatus.Pending && proposal?.status !== ProposalStatus.Active}
+          onClick={() => updatePageState({ confirmDeletion: true })}
+        >
+          {t('page.proposal_detail.delete')}
+        </Button>
+      )}
+      {isCommittee && (proposal?.status === ProposalStatus.Passed || proposal?.status === ProposalStatus.Enacted) && (
```

Comment:
I'd move all these conditions to some boolean variables so they are easier to read and understand what is being rendered.
`showDeleteButton`, `showEnactButton`, `showPassButton`